Lab1b/lexic.txt

```diff
@@ -28,22 +24,11 @@ CHARACTER = ‘LETTER’ | ‘DIGIT’
 STRING = “{LETTER | DIGIT}”
 
 3.	Integer - rule:
-<<<<<<< HEAD
 NUMBER = "0" | NONZERODIGITS[{DIGIT}]
 NONZERODIGITS = “1” | ... | “9”
-SIGNNUMBER = “+”NUMBER | “-“NUMBER | NUMBER
+SIGNNUMBER = [(“+” | “-“)] NUMBER
```

Comment:
I don't think that you need to group the + and - signs. I think that () can be removed.